src/OrchardCore.Modules/OrchardCore.Title/Drivers/TitlePartDisplay.cs

```diff
@@ -21,13 +21,20 @@ public TitlePartDisplay(IStringLocalizer<TitlePartDisplay> localizer)
 
         public override IDisplayResult Display(TitlePart titlePart, BuildPartDisplayContext context)
         {
-            return Initialize<TitlePartViewModel>(GetDisplayShapeType(context), model =>
+            var settings = context.TypePartDefinition.GetSettings<TitlePartSettings>();
+            if (settings.RenderTitle)
             {
-                model.Title = titlePart.ContentItem.DisplayText;
-                model.TitlePart = titlePart;
-            })
-            .Location("Detail", "Header:5")
-            .Location("Summary", "Header:5");
+                return Initialize<TitlePartViewModel>(GetDisplayShapeType(context), model =>
+               {
+                   model.Title = titlePart.ContentItem.DisplayText;
+                   model.TitlePart = titlePart;
+               })
+               .Location("Detail", "Header:5")
```

Comment:
What is the display type that is used in the Container case in the admin? If it's Detail we might have an issue with the ComonMetadata shape that renders a <title> value with display text. (unrelated).
Should we have different settings for the front-end and the backend (not sure we are even using distinct display types)?

Reply:
I didn't change Container def in this PR.  Container doesn't have Title yet.

Reply:
@sebastienros RenderTitle is for front-end. For Admin "Edit" shape Editable and EditableRequired is used

Reply:
>  What is the display type that is used in the Container case in the admin? If it's Detail we might have an issue with the ComonMetadata shape that renders a <title> value with display text. (unrelated).

For Widgets (Detail) we have a `Widget-ContentsMetadata.cshtml` (in the flow module) which provides an empty shape.

For BagParts (in Detail) it remains an unrelated issue